test/e2e/nstemplatetier_test.go

```diff
@@ -297,7 +297,6 @@ func verifyResourceUpdatesForUserSignups(t *testing.T, hostAwait *HostAwaitility
 	for _, usersignup := range userSignups {
 		userAccount, err := memberAwaitility.WaitForUserAccount(usersignup.Status.CompliantUsername,
 			UntilUserAccountHasConditions(Provisioned()),
-			UntilUserAccountHasSpec(ExpectedUserAccount(usersignup.Name, usersignup.Spec.OriginalSub)),
```

Comment:
The `ExpectedUserAccount` function is not used anywhere either, so it could be removed as well. However, this means that the content of the UserSignup (which means the `UserID` and `OriginalSub`) is not verified anywhere, right? Shouldn't we verify these fields?